### DISCUSSION TITLE
Card Source Code?

### DISCUSSION THREAD

**Discussion by onekintaro** [dwainscheeren/dwains-lovelace-dashboard#733] [Dwains Dashboard V3 Alpha]
Hello,
My name is Onekintaro, and I'm a big fan of Dwains Dashboard. 

I wanted to inquire if you would consider releasing the source code of the cards used in Dwains Dashboard. I would like to make some customizations to the cards, but I find it a bit challenging to do so only through the compiled JavaScript code. :)

Thank you for your time

greetings,
Onekintaro

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
